### DISCUSSION TITLE
Stepmania for Batocera?

### DISCUSSION THREAD

**Discussion by LongLiveNoob** [stepmania/stepmania#2243] [General]
Is there any version of Stepmania that works with Batocera?

**Comment by nicksimoneau** [comment]
It could be great to be able to add it to Batocera, I will check to make it in PORTS

**Comment by nicksimoneau** [comment]
> Is there any version of Stepmania that works with Batocera?

If you still interrested I just managed how to get it work with Batocera. Let me know

**Reply by LongLiveNoob** [reply]
Yes, I am still interested

**Comment by nicksimoneau** [comment]
Great let me prepare you the files and I will let you know. Its easy but you need some knowledge in PC/Linux.   
  
You will only have to extract the folder in /roms/Windows   
  
Do you want the base of stepmania only or a package with songs already in?

**Reply by LongLiveNoob** [reply]
It would be great if you add some songs.
Thanks man

**Comment by nicksimoneau** [comment]
Sorry I have ADHD and I totally forgot you!

Here is the zip files for the game.
It's a multi-part zip file so you have to had all the file at the same place to extract it.

Once extracter copy the StepMania.pc folder in /roms/windows

Batocera will need some time the first time to launch the game because it need to configure WINE for the first launch of is live on your PC if you dont have already run once. All the other time it will start as a normal PC game :)

https://mega.nz/folder/adYygAaI#SO29ebApJ5w2Wl5dZB-SKg

Let me know if you have any issue with it :)

**Comment by YourLocalTechGuy** [comment]
if your on a raspberry pi i have recently just manged to make a port of it for batocera on a rpi
i have only tested that it runs on a rpi5 i'm not sure about other models
